torch/_dynamo/variables/user_defined.py

```diff
@@ -140,6 +140,11 @@ def __str__(self):
     def python_type(self):
         return self.value_type
 
+    def as_python_constant(self):
+        if self.value_type is not types.MemberDescriptorType:
+            return self.value
+        return super().as_python_constant()
```

Comment:
nit: special condition + affirmative check + early return reads cleaner

```suggestion
        if self.value_type is types.MemberDescriptorType:
            return super().as_python_constant()
        return self.value
```

Reply:
sounds good. Will do